portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +26,23 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Evelyn!</h1>");
-  }
+    ArrayList<String> favouriteFruits = new ArrayList<>(Arrays. asList("Mango", "Strawberry", "Grapes"));
```

Comment:
`Arrays.asList` already returns a `List`, so you can just change this to:
`List<String> favouriteFruits = Arrays.asList(...);`